docs/topics/contributing-syncing-with-wildfly-swarm-docs.adoc

```diff
@@ -0,0 +1,27 @@
+
+[#syncing-with-wildfly-swarm-docs]
+= Syncing with {WildFlySwarm} Docs
+
+Some documentation files for the {WildFlySwarm} runtim are sourced directly from the {WildFlySwarm} repository.
```

Comment:
"runtim" -> "runtime"

Reply:
Thanks for catching that, I'll fix it in a follow-up pull request.